data/stoa0169a/stoa002/__cts__.xml

```diff
@@ -1,7 +1,8 @@
 <ti:work xml:lang="lat" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:stoa0169a" urn="urn:cts:latinLit:stoa0169a.stoa002">
-  <ti:title xml:lang="eng">Evangeliorum libri quattuor</ti:title>
+  <ti:title xml:lang="lat">Evangeliorum libri quattuor</ti:title>
   <ti:edition workUrn="urn:cts:latinLit:stoa0169a.stoa002" urn="urn:cts:latinLit:stoa0169a.stoa002.opp-lat2">
```

Comment:
edition needs lang attribute

Reply:
hi @lcerrato I think I made the correct change.